### DISCUSSION TITLE
I was about to built openFrameworks from source but... gconf

### DISCUSSION THREAD

**Discussion by whitequill** [openframeworks/openFrameworks#7547] [Q&A]
I just pulled openFrameworks from git then I found that it uses gconf. Why does openFrameworks use a depreciated configuration database system?

**Comment by kylerneva** [comment]
openFrameworks itself doesn’t actively rely on gconf by design. It’s usually pulled in indirectly, through older GTK / GStreamer / system dependencies that OF supports for backward compatibility on Linux.

Even though gconf is deprecated, it may still appear because:
	•	some distros still ship libraries that depend on it
	•	OF aims to stay compatible with a wide range of Linux setups

If you’re on a modern system, gconf is often not actually used at runtime, just linked. Newer setups tend to move toward dconf / GSettings, and this dependency is slowly disappearing as upstream libraries are updated.

So it’s more about legacy system compatibility than an architectural choice by openFrameworks.

Hope that clears it up 👍
